### DISCUSSION TITLE
`Other components` section of `Overview` tab in a translation page would show up to 10 components

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [WeblateOrg/weblate#7286] [Q&A] [answered]
I find that `Other components` section of `Overview` tab in a translation page would show up to 10 components. Is it intended? Or a bug?

**Comment by nijel** [comment, marked as answer]
It's intended for performance reasons, see 2a62e5d3a75, https://github.com/WeblateOrg/weblate/pull/4267, and https://github.com/WeblateOrg/weblate/issues/4220.
